deployer/src/deployment/run.rs

```diff
@@ -272,100 +281,251 @@ impl Built {
             .await
             .map_err(Error::Runtime)?;
 
-        kill_old_deployments.await?;
-        // Execute loaded service
-        load(
+        kill_old_deployments.await?; // TODO: Move to between load and provision? Or even after provision?
```

Comment:
Todo

Reply:
Do you think it should happen after the load phase or after the provision phase?

Reply:
After provisioning... or as late as possible.